File: src/main/scala/cn/whu/ypfamily/EMGeoStreaming/entities/WeatherData.scala
package cn.whu.ypfamily.EMGeoStreaming.entities

import cn.whu.ypfamily.EMGeoStreaming.core.basic.TM_Object

class WeatherData extends Serializable {
  var time: TM_Object = _
  var precipitation: Double = 0.0
}

File: src/main/java/cn/whu/ypfamily/sensorhubclient/entities/WeatherData.java
package cn.whu.ypfamily.sensorhubclient.entities;

public class WeatherData {
	private String sensorId;
	private String precipitation;
	
	public String getPrecipitation() {
		return precipitation;
	}
	public void setPrecipitation(String precipitation) {
		this.precipitation = precipitation;
	}
	public String getSensorId() {
		return sensorId;
	}
	public void setSensorId(String sensorId) {
		this.sensorId = sensorId;
	}
	
	@Override
	public String toString() {
		return "WeatherData [sensorId=" + sensorId + ", precipitation=" + precipitation + "]";
	}
}
